src/ray/raylet/node_manager.cc

```diff
@@ -2647,6 +2647,7 @@ void NodeManager::Stop() {
   // This never fails.
   RAY_CHECK_OK(store_client_.Disconnect());
   object_manager_.Stop();
+  local_object_manager_.DestroyExternalStorage();
```

Comment:
I think this needs to be blocking because we shutsdown the io thread and io service after this